src/main/archetype/aem-experience-template/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "${appId}-template",
```

Comment:
Can we also name the created template folder by the name of the template and ensure that the template it builds really is a ZIP file named that way as well?
`src/main/archetype/aem-experience-template/` => `src/main/archetype/__appId__-template/`

Reply:
I changed the aem-experience-template folder to __appId__-template and did the necessary changes to facilitate that